.github/workflows/docs.yml

```diff
@@ -0,0 +1,71 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - main
+      - feature/nimibook-branch2
```

Comment:
Are the docs also deployed when a pull request is created? If so, that would be a giant security issue.